kingpin/actors/rightscale/deployment.py

```diff
@@ -37,6 +37,15 @@ class DeploymentBaseActor(base.RightScaleBaseActor):
 
     """Abstract Deployment Actor that provides some utility methods."""
 
+    @gen.coroutine
```

Comment:
this really just calls `find_by_name_nad_keys()` the api module has a retry decorator there